source/docs/software/examples-tutorials/drivesim-tutorial/drivetrain-model.rst

```diff
@@ -0,0 +1,132 @@
+Step 2: Creating a Drivetrain Model
+===================================
+In order to accurately determine how your physical drivetrain will respond to given motor voltage inputs, an accurate model of your drivetrain must be created. This model is usually created by measuring various physical parameters of your real robot. In WPILib, this drivetrain simulation model is represented by the ``DifferentialDrivetrainSim`` class.
+
+Creating a ``DifferentialDrivetrainSim`` from Physical Measurements
+-------------------------------------------------------------------
+One way to creating a ``DifferentialDrivetrainSim`` instance is by using physical measurements of the drivetrain and robot -- either obtained through CAD software or real-world measurements. This constructor takes the following parameters:
+
+ * The type and number of motors on one side of the drivetrain.
+ * The gearing between the motors and the wheels as output over input (this number is usually greater than 1 for drivetrains).
+ * The moment of inertia of the drivetrain (this can be obtained from a CAD model of your drivetrain. Usually, this is between 3 and 8 kg m^2).
+ * The mass of the drivetrain (it is recommended to use the mass of the entire robot itself, as it will more accurately model the acceleration characteristics of your robot for trajectory tracking).
+ * The radius of the drive wheels.
+ * The track width (distance between left and right wheels).
+ * Standard deviations of measurement noise: this represents how much measurement noise you expect from your real sensors. The measurement noise is an array with 7 elements, with each element representing the standard deviation of measurement noise in x, y, heading, left velocity, right velocity, left position, and right position respectively. This option can be omitted in C++ or set to ``null`` in Java if measurement noise is not desirable.
```

Comment:
I'm reading alongside the documentation, and my current question is "How do I find out how much measurement noise I expect?"